### PR TITLE
feat: gasleft condition for sstore

### PR DESCRIPTION
**Pull request by MarkuSchick**
fixes https://github.com/smlxl/evm.codes/issues/267

### CODE REVIEW

**Review by peroket — COMMENTED** [review]
Thank you for your contribution :-)

Just two notes: 'gasleft' is more of a coding variable name, I would rephrase to something like 'when the amount of gas left to the transaction is less than or equal 2300'. The second point is that this is true only since the Istanbul revision, so we need to mention that as well.

**Comment by MarkuSchick** [comment]
> Istanbul

Thanks for the great feedback. I use evm.codes every day and am happy to contribute :)

Have fixed it here https://github.com/smlxl/evm.codes/pull/268/commits/bd5005f74d8bf88990bf31849bddc5d64c14f69c

**Review by peroket — APPROVED** [review]
Thank you very much :-)
